lms/static/sass/discussion/_discussion.scss

```diff
@@ -143,15 +143,15 @@ body.discussion {
   }
 
   .wmd-button {
-    @include padding-right(3px);
-    @include padding-left(3px);
     position: absolute;
     display: inline-block;
     width: 20px;
     height: 20px;
     background: none;
     list-style: none;
     cursor: pointer;
+    border: none;
```

Comment:
According to the (semi-)standard that talbs linked to, the order is: border, background, padding, height:

https://github.com/brandon-rhodes/Concentric-CSS/blob/master/style.css#L8

I have to say that your order makes more sense, since the width/height encompass the padding. I'll move ``border`` before ``background`` but I won't reorder everything here.